todoapp/src/components/Todo.vue

```diff
@@ -1,40 +1,210 @@
 <!-- eslint-disable vue/multi-word-component-names -->
-
 <script setup lang="ts">
-import type { TextString } from '../types/text'
+import type { TodoTypes } from '../types/text'
+import Incomplete from '../assets/eclipseblack.svg'
+import Complete from '../assets/completedTask.png'
 
 const props = defineProps<{
-  todos: TextString[]
-}>()
-const emits = defineEmits<{
-  (e: 'removeTodo', index: number): void
+  todos: TodoTypes[]
+  isMobile: boolean
 }>()
 
-function removeTodo(index: number) {
-  emits('removeTodo', index)
+function handleImportance(priority: number) {
+  if (priority === 0) {
+    return 'low'
+  } else if (priority === 1) {
```

Comment:
First of all you don't need to use else if you already have a return in the if but even a better solution could be if you has an object
`const priority = {
    0: 'low',
    1: 'medium',
    2: 'high'
}`
and than the function will be just returning the index from the object
`function handleImportance(index: keyof typeof priority) {
    priority[index]
}`